packages/@aws-cdk/core/test/test.synthesis.ts

```diff
@@ -1,4 +1,5 @@
 import cxapi = require('@aws-cdk/cx-api');
+import cxversioning = require('@aws-cdk/cx-api/lib/versioning');
```

Comment:
why do you need a deep import here? We should export `CLOUD_ASSEMBLY_VERSION` from `@aws-cdk/cx-api`